src/mandelbrot.rs

```diff
@@ -0,0 +1,104 @@
+// The Computer Language Benchmarks Game
+// http://benchmarksgame.alioth.debian.org/
+//
+// contributed by Matt Watson
+// contributed by TeXitoi
+
+use std::io::Write;
+use std::thread;
+const THREADS: usize = 20;
+const MAX_ITER: usize = 50;
+const VLEN: usize = 8;
+const ZEROS: Vecf64 = [0.0; VLEN];
+
+pub type Vecf64 = [f64; VLEN];
```

Comment:
Yeah, I did play around with that a bit, it creates more mess than it saves in something this short though.